src/TypeParser/PhpTypeParser.php

```diff
@@ -111,7 +111,7 @@ private function createType(string $rawType, bool $nullable, \ReflectionClass $r
     private function resolveClass(string $className, \ReflectionClass $reflClass = null): string
     {
         // leading backslash means absolute class name
-        if ('\\' === substr($className, 0, 1)) {
+        if (0 === strpos($className, '\\')) {
```

Comment:
`\strpos`

Reply:
Why? `strpos` and `array_intersect` are not optimized functions: https://github.com/php/php-src/blob/8afa2c632b872bb5c34641d4f67d8543cb82ea0e/Zend/zend_compile.c#L3657-L3736

Reply:
My bad! I thought any function call within a object method should be prefixed so that is skips looking for that function within the class/namespace.